pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -99,16 +100,16 @@ buildPythonApplication rec {
   CGO_ENABLED = 0;
   GOFLAGS = "-trimpath";
 
-  configurePhase = let
-    goModules = (buildGoModule {
+  go-modules =(buildGoModule {
```

Comment:
```suggestion
  go-modules = (buildGoModule {
```

Reply:
Fixed. I look forward to a formatting standard...